service/application/src/main/resources/application.yml

```diff
@@ -68,21 +68,48 @@ eventPortal:
     id: gateway
     name: evmr1
     messaging:
-      standalone: true
+      standalone: false
       rtoSession: false
       enableHeartbeats: false
       testHeartbeats: false
       connections:
         - name: eventPortalGateway
-          authenticationType: ${EP_GATEWAY_AUTH}
-          url: ${EP_GATEWAY_URL}
-          msgVpn: ${EP_GATEWAY_MSGVPN}
+          authenticationType: ${EP_GATEWAY_AUTH:basicAuthentication}
```

Comment:
Please revert this file to the original content.

Reply:
Done